src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Utils/SqidEncoderTests.cs

```diff
@@ -0,0 +1,130 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Utils;
+using Sqids;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests.Utils;
+
+public abstract class SqidEncoderTests
+{
+    private readonly SqidsEncoder<int> _sqidsEncoder = new(new()
+    {
+        Alphabet = "UjIo1rQXDHTmNsy6p4SlAtO0uqhfCiexFMVc5nG3v2Bbz8dKgWJRa79PZYwkEL",
+        MinLength = 5,
+    });
+
+    public class EncodeTests() : SqidEncoderTests
+    {
+        [Fact]
+        public void Success()
+        {
+            var encodedNumber = SqidEncoder.Encode(1);
+
+            var expected = _sqidsEncoder.Encode(1);
+
+            Assert.Equal(expected, encodedNumber);
+        }
+    }
```

Comment:
I agree. I almost didn't mind it at first because I thought if the alphabet/other options ever changed, in either place, I'd want the tests to fail. So it felt OK to me.

But I think hardcoding the hash in there would probably be better so I've changed it